features/index_page.feature

```diff
@@ -0,0 +1,15 @@
+Feature: As a visitor
+So that I can access the site
+I need to be able to see the index page
+
+Background:
+  Given there is a menu called "Lunch"
```

Comment:
You don't need a background for this scenario. Are you using the "Lunch" menu anywhere?